src/app/components/Search/index.tsx

```diff
@@ -52,6 +52,9 @@ const SearchForm = styled('form', {
               display: 'none',
             },
             [`.${inputAdornmentClasses.positionEnd}`]: {
+              'button[type="button"]': {
+                display: 'none',
+              },
```

Comment:
Why is this change necessary?

Reply:
(the first screenshot in the description) 
When mobile footer search input is collapsed, but has a value, a part of it is not fully hidden. Search icon positioning is affected too and only half of icon is visible.